apps/ui/src/components/Modal/Strategies.vue

```diff
@@ -26,7 +26,8 @@ const hasError = ref(false);
 const network = computed(() => getNetwork(props.networkId));
 const filteredStrategies = computed(() => {
   return strategies.value.filter(strategy => {
-    if (props.hiddenStrategies?.includes(strategy.name)) return false;
+    if (props.hiddenStrategies?.includes(strategy.name) || strategy.disabled)
```

Comment:
you remember anything about `hiddenStrategies` ? 🤔 can't find any list. maybe we can deprecate this and add `disabled` on score-api

Reply:
it's here: https://github.com/snapshot-labs/sx-monorepo/blob/d804adbf4a923b3860b34bf9cd7a679ba41ef990/apps/ui/src/components/SetupStrategiesConfiguratorOffchain.vue#L107

To hide the `ticket` strategies on space creation

Reply:
Could we remove `hiddenStrategies` then? as we already get `disabled` property

Reply:
ah this is for space creation, we can keep it